src/app/review-rating/review-rating.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { RequestService } from '@shared/request/request.service';
+
+const api = {
+  post: {
+    reviewRating: 'api/v2/observations/review_rating/create.json',  
+  }
+};
+
+export interface RatingData {
+  assessment_review_id : number;
+  rating : number;
+	comment : string;
```

Comment:
Indentation. 
And there're more wrong indentations in this file, please fix them

Reply:
hum, these identations seems to be fine on my IDE. let me check it.